Library/Formula/libmaxminddb.rb

```diff
@@ -21,6 +21,8 @@ class Libmaxminddb < Formula
 
   depends_on "geoipupdate" => :optional
 
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!